src/components/SingleMovie.js

```diff
@@ -25,10 +27,14 @@ class SingleMovie extends React.Component {
   render() {
     
     const { correctMovie:{title, backdrop_path, tagline, overview, average_rating, release_date, genres, revenue, budget}, error } = this.state;
-    const rating = average_rating ? `${average_rating.toFixed(2)}/10` : null
+    const rating = average_rating && `${average_rating.toFixed(2)}/10`
 
 
-    return error ? <p className="error">Uh oh! Something went wrong</p>: 
+    return error ? 
+    <div className="error-element">
```

Comment:
This will be much clearer for a user if they land on a url that doesn't contain any information